src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,31 @@
+.form-field {
+  font-family: "Cinzel", sans-serif;
```

Comment:
It's better to use a variable for the font family instead of hardcoding it. Consider creating a variable for the font family in a separate file (e.g., '_variables.scss') and importing it.